src/main/java/com/nike/cerberus/client/auth/aws/BaseAwsCredentialsProvider.java

```diff
@@ -80,6 +78,8 @@
 
     public static final MediaType DEFAULT_MEDIA_TYPE = MediaType.parse("application/json; charset=utf-8");
 
+    public static final String IAM_ROLE_ARN_FORMAT = "arn:aws:iam::%s:role/%s";
```

Comment:
What is this used for, this seems like logic that was used for non-sts authentication?

Reply:
removed